### PR TITLE
net-vpn/i2pd: version bump 2.21.0

### PR DESCRIPTION
**Pull request by khumarahn**
This is a no surprise version bump. The only change in the new ebuild is the year in the copyright statement.

Repoman complains:
```
   net-vpn/i2pd/i2pd-2.21.0.ebuild: please migrate from 'versionator' to 'eapi7-ver (built-in since EAPI 7)' on line: 5
```
but from what I understand, eapi7-ver is still testing. So I ignored it for now.

### CODE REVIEW

File: net-vpn/i2pd/i2pd-2.21.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,114 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit eutils systemd user cmake-utils versionator toolchain-funcs
```
Comment (Zlogene) [inline net-vpn/i2pd/i2pd-2.21.0.ebuild]: Where are you using eutils?

File: net-vpn/i2pd/i2pd-2.21.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+SRC_URI="https://github.com/PurpleI2P/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~ia64 ~ppc64 ~x86"
+IUSE="cpu_flags_x86_aes cpu_flags_x86_avx i2p-hardening libressl static +upnp websocket"
+
+# if using libressl, require >=boost-1.65, see #597798
+RDEPEND="!static? ( dev-libs/boost[threads]
+			!libressl? ( dev-libs/openssl:0[-bindist] )
+			libressl? ( dev-libs/libressl:0
```
Comment (Zlogene) [inline net-vpn/i2pd/i2pd-2.21.0.ebuild]: fix your formatting here, please

File: net-vpn/i2pd/i2pd-2.21.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+						>=dev-libs/boost-1.65 )
```
Comment (Zlogene) [inline net-vpn/i2pd/i2pd-2.21.0.ebuild]: you do not need zlib as a dependency here

File: net-vpn/i2pd/i2pd-2.21.0.ebuild (hunk 1 above)
Comment (Zlogene) [inline net-vpn/i2pd/i2pd-2.21.0.ebuild]: replace versionator with eapi7-ver

**Comment by khumarahn** [comment]
@Zlogene @blueness ping? Nothing too important here: a regular version update; I made the changes suggested by @Zlogene.

**Comment by khumarahn** [comment]
@Zlogene @blueness ping?
someone else?
please merge?

**Pull request closed without merging**